code/modules/mapping/space_management/multiz_helpers.dm

```diff
@@ -0,0 +1,47 @@
+/proc/get_step_multiz(ref, dir)
+	if(dir & UP)
+		dir &= ~UP
+		return get_step(SSmapping.get_turf_above(get_turf(ref)), dir)
+	if(dir & DOWN)
+		dir &= ~DOWN
+		return get_step(SSmapping.get_turf_below(get_turf(ref)), dir)
+	return get_step(ref, dir)
+
+/proc/get_dir_multiz(turf/src, turf/other)
+	src = get_turf(src)

```

Comment:
What the fuck kevin

Reply:
What the fuck kevin

Reply:
What the fuck kevin

Reply:
What the fuck kevin, but really, I'm not a coder, but I assume he fucked up and didn't checked his shit again

Reply:
What the fuck kevin